src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloudTest.java

```diff
@@ -221,6 +221,11 @@ public KubernetesClient connect()  {
         Collection<NodeProvisioner.PlannedNode> plannedNodes = cloud.provision(test, 200);
         assertEquals(200, plannedNodes.size());
 
+        cloud.setContainerCapStr("0");
```

Comment:
I guess it goes with https://github.com/jenkinsci/kubernetes-plugin/pull/824/files#r479202733 ?

Reply:
yes, it does